randip/main.go

```diff
@@ -20,12 +20,17 @@ func printHelp() {
 
 Description:
   Generates random IP/IPs.
-  By default, generates 1 random IPv4 address, but can be configured to generate
-  any number of a combination of IPv4/IPv6 addresses.
-  
-  $ %[1]s randip
-  $ %[1]s randip --ipv6 --count 5
-  $ %[1]s randip -4 -n 10
+  By default, generates 1 random IPv4 address with starting range 0.0.0.0 and 
```

Comment:
Please apply the fixes I posted above for the previous help message over here too.